packages/dd-trace/src/telemetry/index.js

```diff
@@ -7,6 +7,7 @@ const { sendData } = require('./send-data')
 const { errors } = require('../startup-log')
 const { manager: metricsManager } = require('./metrics')
 const logs = require('./logs')
+const log = require('../log')
```

Comment:
lol

Reply:
not very readable: at a quick glance i have no idea what's the difference between the two :/